src/ergodex-cdk/components/NetworkDropdown/NetworkDropdown.less

```diff
@@ -1,42 +1,48 @@
-@import './src/assets/styles/variables/colors.less';
-@import './src/assets/styles/variables/gradients.less';
+@import './src/assets/styles/styles';
 
-.NetworkDropdown {
-   &__token-icon {
-      margin-right: 8px;
-  }
+.ant-btn {
+  &.NetworkDropdown {
+    &__container {
+      display: flex;
+      height: 40px;
+      align-items: center;
+      padding-right: 16px;
+      padding-left: 16px;
+      border: 1px solid transparent;
```

Comment:
This causes the following:

<img width="428" alt="Screenshot 2021-10-18 at 19 55 06" src="https://user-images.githubusercontent.com/43601702/137775066-00b2d395-05df-4723-a5b9-f527f98e027d.png">